plugins/c8y_remote_access_plugin/Cargo.toml

```
@@ -0,0 +1,46 @@
+[package]
+name = "c8y-remote-access-plugin"
```

The `package.description` field should be defined on each output binary.

```suggestion
name = "c8y-remote-access-plugin"
description = "Thin-edge.io plugin for the Cumulocity IoT's Cloud Remote Access feature"
```